.github/.wordlist.txt

```diff
@@ -9,6 +9,7 @@
 22
 AAAA
 aarch
+abba
```

Comment:
abba is not a word.

use backticks to escape things that are not words, to prevent spellchecker to try to spell check them